fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergMetadata.java

```diff
@@ -605,14 +605,15 @@ private void collectTableStatisticsAndCacheIcebergSplit(Table table, ScalarOpera
 
             icebergScanTasks.add(icebergSplitScanTask);
 
-            String filePath = icebergSplitScanTask.file().path().toString();
-            if (!filePaths.contains(filePath)) {
```

Comment:
I cannot fully understand this PR. Which part is time consuming?

Reply:
the contains and add operations of hashset is time-consuming.